src/FlaUI.Core.UITests/Elements/MenuTests.cs

```diff
@@ -53,5 +64,25 @@ public void TestMenuWithSubMenusByName()
             Assert.That(fancy, Is.Not.Null);
             Assert.That(fancy.Properties.Name.Value, Is.EqualTo("Fancy"));
         }
+        
+        [Test]
+        public void TestCheckedMenuItem()
+        {
+            if (ApplicationType == TestApplicationType.WinForms)
+            {
+                return;

```

Comment:
You should use `Assert.Ignore` like in `TextBoxTests.TextBoxColorTest`

Reply:
Done.